src/controllers/default.js

```diff
@@ -0,0 +1,17 @@
+// TODO confirm which version of UUID to use
+const uuid4 = require('uuid/v4')
+
+module.exports = function (request, reply) {
+  const context = {
+    pageTitle: 'Waste Permits'
+  }
+
+  // Generate a token
+  const token = uuid4()
+  console.log('Generated token: ' + token)
+
+  reply
+    .view('index', context)
+    .unstate('session')                   // Delete existing session cookie (if there is one)
```

Comment:
Agreed that we wouldn't necessarily want to clear out the session cookie every time we visit the application home page. I envisage some kind of 'Start' button that the user would press to begin an application, so perhaps any existing cooke could be cleared when they click the start button?

I have added a TODO comment to review this later.